.pipelines/templates/build-analyzer.yml

```diff
@@ -16,9 +15,9 @@ steps:
       workingDirectory: 'presidio-analyzer'
       script: |
         set -eux  # fail on error
-        pipenv install --deploy --dev
-        pipenv run python -m spacy download en_core_web_lg
-        pipenv run python -m spacy download en_core_web_sm
+        poetry install --all-extras
```

Comment:
Not for this PR, but maybe we should create another step where the analyzer is installed without extras. It could be that something is failing on the base package but we don't know about it because we install all extras (for example, if we accidentally load a package in one of the files which isn't installed by default.

Reply:
Here since we run all the tests and markers I chose to install all, we have some stanza and transformers tests right?

Reply:
Yes, but they are supposed to be skipped if stanza/transformers are not installed

Reply:
In the CI I assume we want to install and test everything, you mean that we should do another step that tests just the base packages? might make sense yes

Reply:
Yes. Not for this PR though.